docs/classes.list.js

```diff
@@ -574,17 +585,16 @@ export const textAlign = [
 export const textColor = [
   'text-transparent',
   'text-current',
-  's-text-default',
+  's-text',
   's-text-subtle',
   's-text-placeholder',
   's-text-inverted',
   's-text-inverted-subtle',
   's-text-link',
-  's-text-link-hover',
-  's-text-link-active',
-  's-text-link-disabled',
```

Comment:
We have these classes (You can check [here](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=888578831) and it is not in semantic.yml anymore)